### PR TITLE
Added channel life cycle node js sample

### PR DESCRIPTION
**Pull request by VaraPrasad-MSFT**

### CODE REVIEW

File: samples/graph-channel-lifecycle/nodejs/Readme.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    ```
+
+1) Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http -host-header=rewrite 3978
+    ```
+
+
+1) Update the `.env` configuration for the bot to use the ```Microsoft App Id``` and ```App Password``` from the Bot Framework registration. (Note the App Password is referred to as the "client secret" in the azure portal and you can always create a new client secret anytime.)
```
Comment (Wajeed-msft) [inline samples/graph-channel-lifecycle/nodejs/Readme.md:35]: This sample does not use Bot, could you please correct this?

File: samples/graph-channel-lifecycle/nodejs/Readme.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Channel life cycle using Node js
+
```
Comment (Wajeed-msft) [inline samples/graph-channel-lifecycle/nodejs/Readme.md:2]: Please add link to this sample from root readme file.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: samples/graph-channel-lifecycle/nodejs/Readme.md (hunk 1 above)
Comment (VaraPrasad-MSFT) [reply samples/graph-channel-lifecycle/nodejs/Readme.md]: corrected.

File: samples/graph-channel-lifecycle/nodejs/Readme.md (hunk 2 above)
Comment (VaraPrasad-MSFT) [reply samples/graph-channel-lifecycle/nodejs/Readme.md:2]: added link to root read me file.

**Review by asnow003 — APPROVED** [review]

**Pull request merged**